release-notes/ugandaemr-310.md

```diff
@@ -44,15 +54,9 @@ For new machines with no UgandaEMR installed
 
 **For existing versions of UgandaEMR 2.0.x to 3.1.0**
 
-* 32-bit upgrade files- [https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr\_upgrade\_from\_2.0.x\_to\_3.1.0\_32bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_2.0.x_to_3.0.4_32bit.exe/download)
-* 64-bit upgrade files- [https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr\_upgrade\_from\_2.0.x\_to\_3.1.0\_64bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_2.0.x_to_3.0.4_64bit.exe/download)
-
-**For existing versions of UgandaEMR 1.x to 3.1.0**
-
-* 32-bit upgrade files - [https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr\_upgrade\_from\_1.x\_to\_3.1.0\_32bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_1.x_to_3.0.4_32bit.exe/download)
-* 64-bit upgrade files - [https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr\_upgrade\_from\_1.x\_to\_3.1.0\_64bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_1.x_to_3.0.4_64bit.exe/download)
-
-## Fixing Failed Installations and Upgrades
+* 32-bit upgrade files- [https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_2.0.x_to_3.1.0_32bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_2.0.x_to_3.0.4_32bit.exe/download)
+* 64-bit upgrade files- [https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_2.0.x_to_3.1.0_64bit.exe/download](https://sourceforge.net/projects/ugandaemr/files/3.1.0/ugandaemr_upgrade_from_2.0.x_to_3.0.4_64bit.exe/download)
 
-None
+###NOTE 
 
+* When upgrading systems, use the Upgrade option provided for within UgandaEMR.. 
```

Comment:
Change to  "When upgrading to from 3.0.4 or 3.0.1 to 3.1.0 use the Upgrade option (Upgrade War File) found on the Start menu under All Programs/All Apps, UgandaEMR. If the option is not available, Download the update Scripts exe  and run it to fix the upgrade option https://sourceforge.net/projects/ugandaemr/files/3.1.0/update-scripts-3.1.0.exe/download"